src/main/java/seedu/address/model/person/AppointmentDate.java

```diff
@@ -14,7 +15,8 @@
 public class AppointmentDate implements Comparable<AppointmentDate> {
 
     public static final String MESSAGE_CONSTRAINTS_FORMAT =
-            "Appointment Date should follow the format of [mm/dd/yyyy] or [dd-mm-yyyy] or [yyyy-mm-dd]";
+            "Appointment Date should follow the format of "
+                    + "[mm/dd/yyyy HH:mm] or [dd-mm-yyyy HH:mm] or [yyyy-mm-dd HH:mm]";
```

Comment:
minor UX nit, since months (mm) and minutes (mm) are represented identically, would this possibly confuse the user?